include/audio.h

```diff
@@ -1,8 +1,8 @@
 #ifndef H_AUDIO
 #define H_AUDIO
 
-#include <AL/al.h>
-#include <AL/alc.h>
+#include <OpenAL/al.h>
+#include <OpenAL/alc.h>
```

Comment:
Should standardize this. Linux prefers it one way, OSX framework another. I bet Windows does it differently too.